packages/circuit-ui/cli/migrate/__testfixtures__/theme-border-radius.output.js

```diff
@@ -0,0 +1,12 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+const elementStyles = ({ theme }) => css`
+  border-top-left-radius: ${theme.borderRadius.bit};
+  border-top-right-radius: ${theme.borderRadius.byte};
+  border-bottom-left-radius: ${theme.borderRadius.byte};
+  border-bottom-right-radius: ${theme.borderRadius.kilo};
+  border-radius: ${'1px'};
```

Comment:
We're replacing the `borderRadius.kilo` by a hardcoded `'1px'`. Inside Circuit for example, this is used as an edge-case for fine-styling icons like the `Hamburger` or borders like in the `Sidebar.Aggregator`.

It shouldn't be used to _actually_ apply border radius on elements like cards or buttons, so I think that hardcoding 1px here makes sense. Thoughts?

(FYI I initially thought of codemodding it to `0` but there is probably a good reason why there would be a small radius on elements, so I think that keeping it around makes more sense)

Reply:
Hard-coding makes sense to me 👍🏻